public/framework/catalog_cache/cache_manager.ts

```diff
@@ -68,27 +74,34 @@ export class CatalogCacheManager {
    * Retrieves accelerations cache from local storage.
    * @returns {AccelerationsCacheData} The retrieved accelerations cache.
    */
-  static getAccelerationsCache(): AccelerationsCacheData {
+  static getAccelerationsCache(dataSourceMDSId?: string): AccelerationsCacheData {
```

Comment:
Seems like we want to provide a default empty '' when there's no dataSourceMDSId passed. Then you don't have to check existance as well as using '|| '' ' in the following code.